utils/config.py

```diff
@@ -11,15 +11,6 @@
 RELOAD_INTERVAL = os.getenv("RELOAD_INTERVAL", "0")  # compendium static data reload interval
 ECS_METADATA_ENDPT = os.getenv("ECS_CONTAINER_METADATA_URI")  # set by ECS
 MONSTER_TOKEN_ENDPOINT = os.getenv("MONSTER_TOKEN_ENDPOINT")  # S3: monster tokens
-# slash command test guilds - these only apply in development anyway, so hardcoded
-COMMAND_TEST_GUILD_IDS = (
```

Comment:
Was going to ask you to remove it from the readme but this wasn't even on there lolol